### ISSUE TITLE
some reset errors/bugs when aircrafts are crashed

### ISSUE THREAD

**Issue by cjmdd** [harfang3d/dogfight-sandbox-hg2#83]
When aircraftd are shot down or crashed,  they cannot be reintiailezed in the reset  function  that uses reset_machine_matrix() and rearm_machine(). After, there are some errors are as follows:  plane_states =get_planes_state() in get_observation() i.e., return 'Nonetype' has no attribute 'decoder'

**Comment by muratozbek23** [comment]
Yes, there is such a bug. If you take the altitude information of the plane and reset it when it is 50 meters above sea level, the problem will be solved.
